lib/manager/terragrunt/readme.md

```diff
@@ -0,0 +1,18 @@
+Currently by default, Terragrunt support is limited to Terraform registry sources and GitHub sources that include semver refs, e.g. like `github.com/hashicorp/example?ref=v1.0.0`.
+
+It is however possible to also use custom [versioning config](../../../docs/usage/configuration-options.md#versioning) in order to support non-semver references.
+For example, if you want to reference a tag like `module-v1.2.5`, a block like this would work:
+
+```json
+"terraform": {
+	"versioning": "regex:^((?<compatibility>.*)-v|v*)(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)$"
+}
+```
+
+Fixed Terragrunt versions like the following will receive a PR whenever there is a newer version available:
```

Comment:
It's not clear to me what this "Fixed Terragrunt versions" is talking about.
The word fixed brings to mind things like: "Oh it was broken, but now it works." or "I'm hard-coding a version here".
I don't know how to improve this sentence.

Maybe somebody else has a good suggestion for this sentence?

Reply:
@HonkingGoose the same is also in `terraform` manager. But anyway, to make it more clear I suggest this: `Pinned Terragrunt dependecies like the following will receive a PR whenever there is a newer version available:`

Reply:
Hmm, I'm not sure if the word "pinned" is better. The word pinned is usually used to mean: "Use one this specific version", at least the way dependency management bots use the word like this.

Reply:
And that is the exact case here: you need to pin to a version (e.g use this specific version) in order for `renovate` to work and suggest new version available

Reply:
> And that is the exact case here: you need to pin to a version (e.g use this specific version) in order for renovate to work and suggest new version available

---

```suggestion
Pinned Terragrunt dependencies like the the following will receive a PR whenever there is a newer version available:
```

Let's go with this then. :+1:
I fixed a small spelling error in your comment's suggestion with the word `dependencies`.